src/footer.js

```diff
@@ -0,0 +1,13 @@
+import './header.css';
+
+function Footer() {
+    return (
+      <div>
+      <div className="footer">
+      <span style={{paddingBottom:'10px'}}>Made in India</span>
+      </div>
+      </div>
```

Comment:
```suggestion
      <div>
          <div className="footer">
              <span style={{paddingBottom:'10px'}}>Made in India</span>
          </div>
      </div>
```
Please use indentation throughout.